__tests__/components/editor/ResourceTemplateForm.test.js

```diff
@@ -148,134 +93,144 @@ const responseBody = [{
   }
 }]
 
-const lits =  {formData: [{id: 0, uri: 'http://uri', items: [
-        {content: '12345', id: 0, bnode: {termType: 'BlankNode', value: 'n3-0'}, propPredicate: 'http://predicate'}
-      ], rtId: 'resourceTemplate:bf2'}]}
-const lups = { id: 'id', uri: 'uri', label: 'label' }
-const ld = {
-  "@context": {
-    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
-    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
-    "xsd": "http://www.w3.org/2001/XMLSchema#",
-    "bf": "http://id.loc.gov/ontologies/bibframe/",
-    "bflc": "http://id.loc.gov/ontologies/bflc/",
-    "madsrdf": "http://www.loc.gov/mads/rdf/v1#",
-    "pmo": "http://performedmusicontology.org/ontology/"
-  },
-  "@graph": [
-    {
-      "@id": "n3-0",
-      "@type": "http://id.loc.gov/ontologies/bibframe/Instance",
-      "http://id.loc.gov/ontologies/bibframe/issuance": {
-        "@id": "http://id.loc.gov/vocabulary/issuance/mono"
-      },
-      "http://id.loc.gov/ontologies/bibframe/carrier": {
-        "@id": "http://id.loc.gov/vocabulary/carriers/nc"
-      },
-      "http://id.loc.gov/ontologies/bibframe/responsibilityStatement": "STMT",
-      "http://id.loc.gov/ontologies/bibframe/note": {
-        "@id": "n3-8"
-      }
-    },
-    {
-      "@id": "http://id.loc.gov/vocabulary/issuance/mono",
-      "@type": "http://id.loc.gov/ontologies/bibframe/issuance",
-      "rdfs:label": "single unit"
-    },
-    {
-      "@id": "http://id.loc.gov/vocabulary/carriers/nc",
-      "@type": "http://id.loc.gov/ontologies/bibframe/carrier",
-      "rdfs:label": "volume"
-    },
-    {
-      "@id": "n3-8",
-      "@type": "http://id.loc.gov/ontologies/bibframe/Note",
-      "http://www.w3.org/2000/01/rdf-schema#label": "NOTE"
-    }
-  ]
-}
-
 const rtTest = { resourceURI: "http://id.loc.gov/ontologies/bibframe/Work" }
```

Comment:
I can live w it as is, but .... "rtTest" ?   what is this?  A resource template missing most of the required data?

Reply:
Yes, all it requires for the test is the resourceURI.